flycheck-clj-kondo.el

```diff
@@ -63,7 +63,17 @@ Argument EXTRA-ARGS: passes extra args to the checker."
         (warning line-start "<stdin>:" line ":" column ": " (0+ not-newline) "warning: " (message) line-end)
         (info line-start "<stdin>:" line ":" column ": " (0+ not-newline) "info: " (message) line-end))
        :modes (,mode)
-       :predicate (lambda () (string= ,lang (file-name-extension (buffer-file-name)))))))
+       :predicate (lambda ()
+                    (let ((buffer-file-name (buffer-file-name)))
```

Comment:
buffer-file-name is a variable already, so you can in fact just remove this let entirely.

Reply:
Done